packages/apps-config/src/api/spec/zenlink.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017-2021 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
```suggestion

/* eslint-disable sort-keys */

```

We need to add this line for the type files. I think your last2 linting cleanups cleanup broke the ordering because of this - it does make sense elsewhere, but for types since it is a binary format in the SCALE-encoded blob, the ordering as originally specified is critical.

Reply:
Make sense. Thank you!